lms/static/scripts/frontend_apps/components/StudentsSelector.js

```diff
@@ -0,0 +1,92 @@
+import { createElement } from 'preact';
+import propTypes from 'prop-types';
+
+/**
+ * A student navigation tool which shows an active student select list and a previous and next button to
+ * switch students.
+ *
+ * WIP component.
+ */
+
+export default function StudentsSelector({
+  onSelectStudent,
+  selectedStudentIndex,
+  students,
+}) {
+  // Disable the next button if at the end of the list
+  const hasNextStudent = selectedStudentIndex + 1 < students.length;
+  // Disable the previous button if at the start of the list
+  const hasPrevStudent = selectedStudentIndex > 0;
+
+  /**
+   * Select the next student in the list
+   */
+  const onNextStudent = () => {
+    if (selectedStudentIndex + 1 < students.length) {
+      onSelectStudent(selectedStudentIndex + 1);
+    }
+  };
+  /**
+   * Select the previous student in the list
+   */
+  const onPrevStudent = () => {
+    if (selectedStudentIndex > 0) {
+      onSelectStudent(selectedStudentIndex - 1);
+    }
+  };
+
+  /**
+   * Build the <select> list from the current array of students
+   */
+  const studentList = () => {
```

Comment:
Since this function has no arguments and is used unconditionally, it could just be a value rather than a function.